devlab/tests/scenarios/cold_migrate.yaml

```diff
@@ -106,3 +106,4 @@ process:
       - rename_info_iter: True
   - verify:
       - verify_vms: True
+      - remove_failed_instances: True
```

Comment:
Is it actually `verify` action? Maybe it is better to move it to the `transport_instances_and_dependency_resources` section?